### PR TITLE
User achievements page

### PR DESCRIPTION
**Pull request by s-kashko**

### CODE REVIEW

File: app/containers/Achievements/Achievement.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  return (
+    <Bage>
+      {reached && <Img src={achievementReached} alt="reached achievement" />}
+      {!reached && (
+        <Img src={achievementNotReached} alt="not reached achievement" />
+      )}
+      <div>
+        <TitleBlock>
+          <strong>
+            <ValueSpan>{(id === 1 || id === 2 || id === 3) && value}</ValueSpan>
```
Comment (sergeyi23) [inline app/containers/Achievements/Achievement.jsx:48]: Use constants instead of these magic numbers.

Comment (sergeyi23) [reply app/containers/Achievements/Achievement.jsx:48]: Can setup array in constants and then use `includes(value)` to check that it is one of a values.

**Review by sergeyi23 — APPROVED** [review]
See few minor comments. Otherwise looks good.

File: app/containers/Achievements/Achievement.jsx (hunk 1 above)
Comment (s-kashko) [reply app/containers/Achievements/Achievement.jsx]: badgesWithValuesIds array added for detecting badges with values

**Pull request merged**